.gitignore

```diff
@@ -98,3 +98,7 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Mac OS X stuff
+Thumbs.db
+.DS_Store
```

Comment:
Would you mind removing the existing .DS_Store files that have already been added, too, please?  

`find . -name ".DS_Store" -exec git rm {} \;`